add-on/_locales/en/messages.json

```diff
@@ -169,6 +169,14 @@
     },
     "description": "A generic placeholder for error notification (notify_inlineErrorMsg)"
   },
+  "notify_startIpfsNodeTitle": {
+    "message": "Failed to start IPFS node",
```

Comment:
We should change key names to reflect that message is for a failure:
- `notify_startIpfsNodeTitle`
- `notify_stopIpfsNodeTitle`

Reply:
ha, oops I didn't mean to not indicate it was for a failure!